README.md

```diff
@@ -371,6 +371,21 @@ This will change the prompt to:
 
 ![[master ≡] ~\GitHub\posh-git&#10;02-18 14:04:35 38> ][prompt-custom]
 
+Finally, the path portion of the prompt can be contained within delimiters.
+For instance, if you would like the containing characters to be red, curly braces, the following settings can be used:
+
+```text
+$GitPromptSettings.BeforePath = '{'
+$GitPromptSettings.AfterPath = '}'
+$GitPromptSettings.BeforePath.ForegroundColor = 'Red'
+$GitPromptSettings.AfterPath.ForegroundColor = 'Red'
+```
+
+With these additional values, the previous prompt would become
+
+![[master ≡] {~\GitHub\posh-git}&#10;02-18 14:04:35 38> ][prompt-custom-wpathdelim]
+
+
 ### Prompt Layouts
```

Comment:
Can you update line 399 from:
```
{DPPrefix}{DPPath}{PathStatusSeparator}<{BeforeStatus}{Status}{AfterStatus}>{DPBeforeSuffix}<{DPDebug}><{DPTimingFormat}>{DPSuffix}
```
to
```
{DPPrefix}{BeforePath}{DPPath}{AfterPath}{PathStatusSeparator}<{BeforeStatus}{Status}{AfterStatus}>{DPBeforeSuffix}<{DPDebug}><{DPTimingFormat}>{DPSuffix}
```
and change line 405 to:
```
{DPPrefix}<{BeforeStatus}{Status}{AfterStatus}>{PathStatusSeparator}{BeforePath}{DPPath}{AfterPath}{DPBeforeSuffix}<{DPDebug}><{DPTimingFormat}>{DPSuffix}
```

Reply:
Done